data/software_versions_shared_dedicated.yml

```diff
@@ -0,0 +1,29 @@
+# Example usage:
+#myDB
```

Comment:
It misses Redis and Elasticsearch: 
elasticsearch: 
   dedicated:
       - "v7.17.10"
       - "v8.10.2"
 redis: 
   dedicated:
       - "v7.2.4"